packages/react-portal/src/virtualParent/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './elementContains';
+export * from './setVirtualParent';
+export * from './isVirtualElement';
+export * from './getParent';
+export * from './getVirtualParent';
```

Comment:
nit: Order alphabetically
```suggestion
export * from './elementContains';
export * from './getParent';
export * from './getVirtualParent';
export * from './isVirtualElement';
export * from './setVirtualParent';
```